### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by chisel316** [gt7coder/grandturismo-srs-proxy#1]
Thanks for this awesome app! I have a GT7 profile tuned for my H3 that is working well. I have two suggestions:

1. Please add an option to disable motion during replays.
2. Please add support for traction loss. I see telemetry data for yaw, but not for traction loss. Maybe you can map yaw to traction loss so it will appear in SRS.

**Comment by chisel316** [comment]
After reviewing the code I see there is no flag for in a replay and GT7 doesn't export traction loss directly. While I now realize that mapping yaw to traction loss doesn't work, there is enough data output from the telemetry to calculate it using some clever math.

**Comment by vthinsel** [comment]
Hello,



I'll see if I can find something to disable during replay, but I couldnt figure out one easily so far.

For traction loss, it works nicely on my side. It is called slip angle as you can see in https://www.gtplanet.net/forum/threads/gt7-is-compatible-with-motion-rig.410728/post-13831008

What software are you using ?














---- Le mar., 20 sept. 2022 04:57:44 +0200 danpmarder ***@***.***> a écrit ----





Thanks for this awesome app! I have a GT7 profile tuned for my H3 that is working well. I have two suggestions:

Please add an option to disable motion during replays.

Please add support for traction loss. I see telemetry data for yaw, but not for traction loss. Maybe you can map yaw to traction loss so it will appear in SRS.

**Comment by chisel316** [comment]
I'm using SRS with the proxy here. You're referring to the XSim proxy. Did you write both? The SRS proxy is very simple, while the XSim proxy uses advanced mathematical calculations to transform the data. I actually merged the slip angle code into the SRS proxy locally to get traction loss working.

**Comment by vthinsel** [comment]
I made only XSim proxy, based on SRS proxy that was done by gt7coder https://github.com/gt7coder/grandturismo-srs-proxy

Good news if you got it working. But SRS proxy values are not accurate, as they would need calculation as I did in mine. This is expecially true for acceleration which is provided as world values, and not local values. 

The real stuff goes from that point: https://www.gtplanet.net/forum/threads/gt7-is-compatible-with-motion-rig.410728/post-13824752












---- Le mar., 20 sept. 2022 21:37:08 +0200 danpmarder ***@***.***> a écrit ----





I'm using SRS with the proxy here. You're referring to the XSim proxy. Did you write both? The SRS proxy is very simple, while the XSim proxy uses advanced mathematical calculations to transform the data. I actually merged the slip angle code into the SRS proxy locally to get traction loss working.
